cmd/verify/verify.go

```diff
@@ -44,14 +44,14 @@ type Checksums struct {
 func main() {
 	config := NewConfig()
 	mq := broker.New(config.Broker)
-	db, err := postgres.NewDB(config.Postgres)
+	dbs, err := postgres.NewDB(config.Postgres)
```

Comment:
Unnecessary renaming
```suggestion
	db, err := postgres.NewDB(config.Postgres)
```